packages/blade/src/components/FileUpload/FileUpload.web.tsx

```diff
@@ -0,0 +1,367 @@
+                        }
+                      >
+                        Upload
+                      </Text>
+                    </Box>
+                  </Box>
+                </Box>
+              </StyledFileUploadWrapper>
+            </BaseBox>
+          </SelectorLabel>
+          {isOneFileSelectedWithSingleUpload && (
+            <BaseBox key={selectedFiles[0].id} marginBottom="spacing.5">
+              <FileUploadItem
+                file={selectedFiles[0]}
+                onRemove={() => {
+                  const newFiles = selectedFiles.filter(({ id }) => id !== selectedFiles[0].id);
+                  setSelectedFiles(newFiles);
+                  onRemove?.({ file: selectedFiles[0] });
+                }}
+                onCancel={() => {
+                  const newFiles = selectedFiles.filter(({ id }) => id !== selectedFiles[0].id);
+                  setSelectedFiles(newFiles);
+                  onCancel?.({ file: selectedFiles[0] });
+                }}
+                onPreview={onPreview}
+              />
+            </BaseBox>
+          )}
+        </BaseBox>
+      </BaseBox>
+      {/* the magic number 136 is basically max-width of label i.e 120 and then right margin i.e 16 which is the spacing between label and input field */}
+      {willRenderHintText && (
+        <BaseBox
+          marginLeft={makeSize(label && isLabelLeftPositioned ? 136 : 0)}
+          marginBottom="spacing.5"
+        >
+          <BaseBox display="flex" flexDirection="row" justifyContent="'space-between">
+            <FormHint
+              type={getHintType({
+                validationState: showError ? 'error' : validationState,
+                hasHelpText: Boolean(helpText),
+              })}
+              helpText={helpText}
+              errorText={errorMessage}
+              helpTextId={helpTextId}
+              errorTextId={errorTextId}
+            />
+          </BaseBox>
+        </BaseBox>
+      )}
+      {!isOneFileSelectedWithSingleUpload &&
+        selectedFiles.map((file) => (
+          <BaseBox
+            key={file.id}
+            marginLeft={makeSize(label && isLabelLeftPositioned ? 136 : 0)}
+            marginBottom="spacing.3"
+          >
+            <FileUploadItem
+              file={file}
+              onRemove={() => {
+                const newFiles = selectedFiles.filter(({ id }) => id !== file.id);
+                setSelectedFiles(newFiles);
+                onRemove?.({ file });
+              }}
+              onCancel={() => {
+                const newFiles = selectedFiles.filter(({ id }) => id !== file.id);
+                setSelectedFiles(newFiles);
+                onCancel?.({ file });
+              }}
+              onPreview={onPreview}
+            />
+          </BaseBox>
+        ))}
+    </BaseBox>
+  );
+};
+
+const FileUpload = assignWithoutSideEffects(forwardRef(_FileUpload), {
+  displayName: MetaConstants.FileUpload,
```

Comment:
It shouldn't be MetaConstants no? we usually have TitleCased names here for displayName and componentId

Reply:
Correct, fixed.